### ISSUE TITLE
OTokenSecurityException: The token provided is expired

### ISSUE THREAD

**Issue by Dollyn** [orientechnologies/orientdb#7690]
### OS: centos7

```
Caused by: com.orientechnologies.orient.enterprise.channel.binary.OTokenSecurityException: The token provided is expired
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.orientechnologies.orient.client.binary.OChannelBinaryAsynchClient.throwSerializedException(OChannelBinaryAsynchClient.java:442)
	at com.orientechnologies.orient.client.binary.OChannelBinaryAsynchClient.handleStatus(OChannelBinaryAsynchClient.java:393)
	at com.orientechnologies.orient.client.binary.OChannelBinaryAsynchClient.beginResponse(OChannelBinaryAsynchClient.java:275)
	at com.orientechnologies.orient.client.binary.OChannelBinaryAsynchClient.beginResponse(OChannelBinaryAsynchClient.java:167)
	at com.orientechnologies.orient.client.remote.OStorageRemote.beginResponse(OStorageRemote.java:2225)
	at com.orientechnologies.orient.client.remote.OServerAdmin$6.execute(OServerAdmin.java:307)
	at com.orientechnologies.orient.client.remote.OServerAdmin$6.execute(OServerAdmin.java:294)
	at com.orientechnologies.orient.client.remote.OServerAdmin.networkAdminOperation(OServerAdmin.java:699)
	at com.orientechnologies.orient.client.remote.OServerAdmin.existsDatabase(OServerAdmin.java:294)
```

## Actual behavior  

OServerAdmin idle for some time, then the new admin request will cause the exception. The client side just closed the session.

```
protected <T> T networkAdminOperation(final OStorageRemoteOperation<T> operation, final String errorMessage) {

    OChannelBinaryAsynchClient network = null;
    try {
      //TODO:replace this api with one that get connection for only the specified url.
      String serverUrl = storage.getNextAvailableServerURL(false, session);
      do {
        try {
          network = storage.getNetwork(serverUrl);
        } catch (OException e) {
          serverUrl = storage.useNewServerURL(serverUrl);
          if (serverUrl == null)
            throw e;
        }
      } while (network == null);


      T res = operation.execute(network, storage.getCurrentSession());
      storage.connectionManager.release(network);
      return res;
    } catch (Exception e) {
      if(network != null)
        storage.connectionManager.release(network);
      storage.close(true, false);
      throw OException.wrapException(new OStorageException(errorMessage), e);
    }
  }
```

After it closed the storage, the following request will cause a new 'invalid session' exception
```
Caused by: com.orientechnologies.common.io.OIOException: Invalid session for URL 'kg1:12424'
	at com.orientechnologies.orient.client.binary.OChannelBinaryAsynchClient.beginRequest(OChannelBinaryAsynchClient.java:471)
	at com.orientechnologies.orient.client.remote.OStorageRemote.beginRequest(OStorageRemote.java:2129)
	at com.orientechnologies.orient.client.remote.OServerAdmin$6.execute(OServerAdmin.java:299)
	at com.orientechnologies.orient.client.remote.OServerAdmin$6.execute(OServerAdmin.java:294)
	at com.orientechnologies.orient.client.remote.OServerAdmin.networkAdminOperation(OServerAdmin.java:699)
```

and this new error info is somewhat misleading.

## Expected behavior  
auto renew the token, or auto reconnect?

**Comment by KishoDai** [comment]
encounted the same issue!

**Comment by tglman** [comment]
Hi @KishoDai, @Dollyn,

Similar issues have already fixed in the 2.2.31 hotfix, you can just update to the las hotfix ( 3.3.34 now).

Closing
